CONTRIBUTING.md

```diff
@@ -233,3 +235,55 @@ _NOTE: with all of these steps, if you have SSH Keys set up, you will want to us
 5. Make a new branch: `git checkout -b your_branch_name`.
 6. Pull in content from you private repo: `git pull private main`.
 7. Push your new work up to GitHub: `git push origin your_branch_name`.
+
+## Troubleshooting
+
+This section attempts to capture some of the more common troubleshooting techniques to try
+if you face issues in contributing.
+
+### General Install, compile or build issues
+
+Many of the common errors you'll face can be resolve by the following:
+
+1. Ensure you are using `Yarn` not `NPM`
+2. When in doubt completely delete the `node_module` folder in the root project directory, and re-run `yarn install`
+3. Always run `git pull` whenever you intend to create a new working branch
+4. Always run `Yarn install` whenever you git pull.
+
+### Build / compile issues on the Apple M1 chipset
+
```

Comment:
You might add just a tiny paragraph that says something like:

> If you are working on an M1 Mac, you may run into an error like the following when installing `node_modules`:

Reply:
added.